cfgov/v1/models/browse_page.py

```diff
@@ -0,0 +1,54 @@
+from wagtail.wagtailcore.fields import StreamField
+from wagtail.wagtailadmin.edit_handlers import TabbedInterface, ObjectList, \
+    StreamFieldPanel
+from wagtail.wagtailcore import blocks
+from wagtail.wagtailcore.models import PAGE_TEMPLATE_VAR
+
+from .base import CFGOVPage
+from . import molecules
+from . import organisms
+
+
+class BrowsePage(CFGOVPage):
+    header = StreamField([
+        ('text_introduction', molecules.TextIntroduction()),
+    ], blank=True)
+
+    content = StreamField([
+        ('image_text_25_75_group', organisms.ImageText2575Group()),
+        ('image_text_50_50_group', organisms.ImageText5050Group()),
+        ('half_width_link_blob_group', organisms.HalfWidthLinkBlobGroup()),
+        ('well', organisms.Well()),
+        ('full_width_text', organisms.FullWidthText()),
+        ('expandable_group', organisms.ExpandableGroup()),
+        ('table', organisms.Table()),
+    ], blank=True)
+
+    side_navigation = StreamField([], blank=True)
+
+
+    # General content tab
+    content_panels = CFGOVPage.content_panels + [
+        StreamFieldPanel('header'),
+        StreamFieldPanel('content'),
+    ]
+
+    sidebar_panels = [
+                         StreamFieldPanel('side_navigation'),
+                     ] + CFGOVPage.sidefoot_panels
+
+    # Tab handler interface
+    edit_handler = TabbedInterface([
+        ObjectList(content_panels, heading='General Content'),
+        ObjectList(CFGOVPage.sidefoot_panels, heading='Sidebar'),
+        ObjectList(CFGOVPage.settings_panels, heading='Configuration'),
+    ])
+
+    def get_context(self, request, *args, **kwargs):
+        return {
+            PAGE_TEMPLATE_VAR: self,
+            'self': self,
+            'request': request,
+        }
```

Comment:
This is duplicate from the Wagtail code. https://github.com/torchbox/wagtail/blob/master/wagtail/wagtailcore/models.py#L648